data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -60,9 +50,9 @@
 {% elif table_ref.table_id == 'death' %}
 WHERE COALESCE(death_date, EXTRACT(DATE FROM death_datetime)) >= d.deactivated_date
 {% elif table_ref.table_id in ['drug_era', 'condition_era', 'dose_era', 'payer_plan_period']  %}
```

Comment:
Do you think this will work? I took the conditions from the fitbit CR you added.
```suggestion
WHERE COALESCE(death_date, EXTRACT(DATE FROM death_datetime)) >= d.deactivated_date
{% elif table in ['activity_summary', 'heart_rate_summary'] %}
WHERE date >= d.deactivated_date
{% elif table in ['heart_rate_minute_level', 'steps_intraday']  %}
WHERE datetime >= PARSE_DATETIME('%F', CAST(d.deactivated_date as STRING))
{% elif table_ref.table_id in ['drug_era', 'condition_era', 'dose_era', 'payer_plan_period']  %}
```